rllib/tuned_examples/bc/cartpole_bc.py

```diff
@@ -49,7 +49,12 @@
     # as remote learners.
     .offline_data(
         input_=[data_path],
-        input_read_method_kwargs={"override_num_blocks": max(args.num_gpus, 1)},
+        input_read_method_kwargs={"override_num_blocks": max(args.num_gpus, 2)},
```

Comment:
Can we add a one-line comment to each of these settings, explaining, why we have to set these and what they do under the hood?